Yuneec_SDK_iOS/YNCSDKCamera.mm

```diff
@@ -95,6 +186,26 @@ + (void)stopPhotoIntervalWithCompletion:(YNCCameraCompletion)completion {
     dc->device().camera().stop_photo_interval_async(std::bind(&receive_camera_result, completion, _1));
 }
 
+//MARK: set camera mode
++ (void) setCameraMode:(YNCCameraMode)cameraMode WithCompletion:(YNCCameraModeCompletion)completion {
+    DroneCore *dc = [[YNCSDKInternal instance] dc];
+    Camera::Mode cameraModeVal = getCameraModeEnum(cameraMode);
+    dc->device().camera().set_mode_async(cameraModeVal,std::bind(&receive_camera_mode_result,completion,_1,_2));
```

Comment:
I prefer space after `,`, and if needed newlines.

Reply:
right! Sure.

Reply:
right! sure